src/test/java/no/unit/nva/database/DatabaseServiceImplTest.java

```diff
@@ -39,37 +46,128 @@ public void getUserThrowsIllegalStateExceptionWhenItReceivesInvalidUserFromDatab
         UserDb userWithoutUsername = new UserDb();
         userWithoutUsername.setInstitution(SOME_INSTITUTION);
 
-        PaginatedQueryList<UserDb> response = mockResponseFromDynamoMapper(userWithoutUsername);
-        DynamoDBMapper mockMapper = mockDynamoMapperReturningInvalidUser(response);
-        DatabaseService service = new DatabaseServiceImpl(mockMapper);
+        DatabaseService service = mockServiceReceivingInvalidUserDbInstance();
 
         Executable action = () -> service.getUser(someUser);
         IllegalStateException exception = assertThrows(IllegalStateException.class, action);
 
-        String expectedMessageContent = DatabaseServiceImpl.INVALID_USER_IN_DATABASE;
+        String expectedMessageContent = DatabaseServiceImpl.INVALID_ENTRY_IN_DATABASE_ERROR;
         assertThat(exception.getMessage(), containsString(expectedMessageContent));
     }
 
     @Test
-    public void getUserThrowsNotFoundExceptionWhenUserDoesNotExist() {
-        DatabaseService service = new DatabaseServiceImpl(mapper);
+    public void getRoleThrowsIllegalStateExceptionWhenItReceivesInvalidUserFromDatabase() {
+
+        DatabaseService service = mockServiceReceivingInvalidRoleDbInstance();
+
         Executable action = () -> service.getUser(someUser);
-        NotFoundException exception = assertThrows(NotFoundException.class, action);
-        assertThat(exception.getMessage(), containsString(DatabaseServiceImpl.USER_NOT_FOUND_MESSAGE));
+        IllegalStateException exception = assertThrows(IllegalStateException.class, action);
+
+        String expectedMessageContent = DatabaseServiceImpl.INVALID_ENTRY_IN_DATABASE_ERROR;
+        assertThat(exception.getMessage(), containsString(expectedMessageContent));
+    }
+
+    private DatabaseService mockServiceReceivingInvalidUserDbInstance() {
```

Comment:
Probably move the private methods to the bottom

Reply:
fixed